Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -398,17 +398,51 @@ private static string ConvertLocalToGcsFolderPath(string localFilePath)
         }
     }
 
+    // TODO(chrsmith): Support iterating through the result prefixes as well as the items.
+    // This is necessary to see the "subfolders" in Cloud Storage, even though the concept
+    // does not exist.
+
     /// <summary>
     /// <para type="synopsis">
-    /// Get-GcsObject returns the Google Cloud Storage Object metadata with the given name. (Use
-    /// Find-GcsObject to return multiple objects or Read-GcsObject to get its contents.)
+    /// Get-GcsObject returns Google Cloud Storage Objects and their metadata.
+    /// (Use Read-GcsObject to get its contents.)
+    /// </para>
+    /// <para type="description">
+    /// Given a Google Cloud Storage Bucket, returns Google Cloud Storage Objects and their metadata.
     /// </para>
     /// <para type="description">
-    /// Returns the give Storage object's metadata.
+    /// If no parameter besides -Bucket is provided, all objects in the bucket are returned.
+    /// If a given prefix string is provided, returns all Cloud Storage objects identified
+    /// by the prefix string.
+    /// </para>
+    /// <para type="description">
+    /// An optional delimiter may be provided. If used, will return results in a
+    /// directory-like mode, delimited by the given string. e.g. with objects "1,
```

Comment:
The description and example are a little hard to follow.

Is this well documented for GCS?

Can you add a more detailed example?